### PR TITLE
Reset Hasura metadata for graphql-engine v2 (again)

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues
Originally reviewed in https://github.com/cityofaustin/atd-moped/pull/736.

This PR resets our Hasura metadata `tables.yaml` to match Hasura's ordering in graphql-engine v2.

This change is the result of running `hasura metadata export` against the DB.

This PR also resolves https://github.com/cityofaustin/atd-data-tech/issues/9643 by commiting `network.yaml` to version control.

## Testing
**URL to test:** local or https://moped-test.austinmobility.io/moped
**Steps to test:**
This is tech debt only. It would be good to kick the tires on basic Moped CRUD operations to make sure nothing looks amiss.

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

**Comment by johnclary** [comment]
OK—let's try this again. I checked the yaml diff and it's the same as last time: a handful of adds/deletes of default values for `backend_only` and `insertion_order`. 

Happy to test further but I'm ready to merge once y'all toss me an approval.

**Review by mddilley — APPROVED** [review]
🙏 🚀🚀🚀

**Pull request merged**
